_toc.yml

```diff
@@ -1,5 +1,59 @@
 - file: index
 - chapters:
+  - file: tutorials/start_index
+    sections:
+    - file: tutorials/fundamentals/installation
+    # - file: tutorials/absolute_beginners
+    - file: tutorials/fundamentals/getting_started
+    - file: plugins/stable/find-and-install-plugin
+    - file: tutorials/fundamentals/viewer
+    # The following will be replaced by
+    # - file: tutorials/fundamentals/layers/index
+    - file: tutorials/fundamentals/image
+    - file: tutorials/fundamentals/labels
+    - file: tutorials/fundamentals/points
+    - file: tutorials/fundamentals/shapes
+    - file: tutorials/fundamentals/surface
+    - file: tutorials/fundamentals/tracks
+    - file: tutorials/fundamentals/vectors
+    title: Getting started
+  - file: tutorials/index
+    sections:
+    - file: tutorials/segmentation/index
+    - file: tutorials/tracking/index
+    - file: tutorials/applications/annotate_points
+    - file: tutorials/applications/dask
+    # - file: tutorials/particle_trackpy
+    - file: tutorials/further-resources/napari-workshops
+    title: Tutorials
+  - file: howtos_index
+    sections:
+    # - file: tutorials/fundamentals/layers/index
+    - file: tutorials/applications/napari_imageJ
+    - file: guides/stable/perfmon
+    - file: guides/stable/docker
+    - file: guides/stable/connecting_events
+    # - file: guides/how-to-run-examples
+    # - file: guides/remote-based-desktops
+    title: How-to guides
+  - file: guides/stable/index
+    sections:
+    # - file: guides/layers
+    - file: guides/stable/event_loop
+    - file: guides/stable/threading
+    - file: guides/stable/rendering-explanation
+    - file: guides/stable/rendering
+    - file: guides/stable/performance
+    - file: guides/stable/magicgui
```

Comment:
In my opinion, magicgui is very important, and we want new people to know about it early on. I'd probably try and put this at the start of this section.